packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -255,6 +263,33 @@ export function RoundApplicationForm(props: {
               />
             </div>
           </form>
+          <InfoModal
```

Comment:
for future reference: 

headless ui doesn't support simultaneous Dialogs, only nested Dialogs -- if there are two Dialogs open at the same time (including animation/transition time), it can cause weird issues like a scroll-locking bug: https://github.com/gitcoinco/grants-round/issues/449#issuecomment-1270723361

we'll have to be careful about wiring together the button actions / cancels for the 3 modals 😬

Reply:
resolved with [635feb4](https://github.com/gitcoinco/grants-round/pull/568/commits/635feb4a3385a844f8fc4cd84ec965059c040150) that nests the three modals